modules/distribution/src/repository/resources/component.xml

```diff
@@ -0,0 +1,105 @@
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+<component xmlns="http://products.wso2.org/carbon">
+	<menus>			
+		<menu>
+			<id>registry_menu</id>
+			<i18n-key>component.governance.registry</i18n-key>
+			<i18n-bundle>org.wso2.carbon.i18n.Resources</i18n-bundle>
+			<parent-menu></parent-menu>
+			<link>#</link>
+			<region>region3</region>
+			<order>26</order>
+			<style-class>home</style-class>
+        </menu>
+		<menu>
+			<id>governance_menu</id>
+			<i18n-key>component.governance</i18n-key>
+			<i18n-bundle>org.wso2.carbon.i18n.Resources</i18n-bundle>
+			<parent-menu></parent-menu>
+			<link>#</link>
+			<region>region3</region>
+			<order>28</order>
+			<style-class>home</style-class>
+        </menu>
+        <menu>
+            <id>extensions_menu</id>
+            <i18n-key>component.extensions</i18n-key>
+            <i18n-bundle>org.wso2.carbon.i18n.Resources</i18n-bundle>
+            <parent-menu></parent-menu>
+            <link>#</link>
+            <region>region3</region>
+            <order>25</order>
+            <style-class>home</style-class>
+        </menu>
+		<!--<menu>
```

Comment:
The default elements in carbon product are commented here, because they are not required for PC at the time. But can be removed. So fixed.